Classes/Plugin/FrequentSearches/FrequentSearches.php

```diff
@@ -83,7 +83,7 @@ protected function performAction()
      * @param \ApacheSolrForTypo3\Solr\Template $template
      * @return \ApacheSolrForTypo3\Solr\Template
      */
-    protected function postInitializeTemplateEngine($template)
+    protected function postInitializeTemplateEngine(\ApacheSolrForTypo3\Solr\Template $template)
```

Comment:
please import FQDN in use as its done in https://github.com/TYPO3-Solr/ext-solr/blob/release-6.1.x/Classes/Plugin/PluginBase.php#L37
`protected function postInitializeTemplateEngine(Template $template)`